.ci/setup.sh

```diff
@@ -14,13 +14,13 @@ source "${cidir}/lib.sh"
 source /etc/os-release
 
 if [ "$ID" == fedora ];then
-	sudo -E dnf -y install automake yamllint coreutils moreutils
+	sudo -E dnf -y install automake yamllint coreutils moreutils bc
```

Comment:
heh heh - we could factor out a bunch of this to 'distro-common' packages by the look of it. I know how much you hate duplication @jodh-intel ;-)